Spec/Tests/RealtimeClientChannelTests.swift

```diff
@@ -3889,6 +3889,57 @@ class RealtimeClientChannelTests: XCTestCase {
             channel.attach()
         }
     }
+    
+    // RTL13b
+    func test__131b__Channel__if_the_channel_receives_a_server_initiated_DETACHED_message_and_if_the_attempt_to_reattach_fails_then_the_channel_will_transition_to_SUSPENDED_state_with_periodic_reattach_with_channelRetryTimeout() {
+        
+        let options = AblyTests.commonAppSetup()
+        options.channelRetryTimeout = 1.0
+        options.autoConnect = false
+        let client = ARTRealtime(options: options)
+        client.internal.setTransport(TestProxyTransport.self)
+        
+        let previousRealtimeRequestTimeout = ARTDefault.realtimeRequestTimeout()
+        ARTDefault.setRealtimeRequestTimeout(1.0)
```

Comment:
I _think_ that this plus the `transport.actionsIgnored += [.attached]` lines are intended to make sure that each attach attempt times out after one second, is that right? Perhaps a comment or two would make this clearer.